volatility3/framework/layers/resources.py

```diff
@@ -1,7 +1,8 @@
 # This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
 #
-
+import s3fs
```

Comment:
Ideally fail gracefully if these can't be imported.  Ideally these wouldn't be part of the `resources.py` since we don't want to stop all of volatility running if they're not found.